codegen-server.yml

```diff
@@ -4,10 +4,7 @@ hooks:
     - prettier --write
 generates:
   ./pages/api/graphql-rest.page.generated.ts:
-    schema:
-      - './pages/api/Schema/ContactFilters/contactFilters.graphql'
-      - './pages/api/Schema/ExportContacts/exportContacts.graphql'
-      - './pages/api/Schema/TaskAnalytics/taskAnalytics.graphql'
+    schema: ./pages/api/Schema/**/*.graphql
```

Comment:
I think the ``` ''``` was causing issues with this. I believe you can use them for specific file paths, but when trying to use them with wildcards it seems to break things.